### ISSUE TITLE
Create tutorial(s) that demonstrate regtools functionality using HCC1395 data

### ISSUE THREAD

**Issue by malachig** [griffithlab/regtools#21]
* Starting with BAM files (DNA and RNA), somatic variant VCF, germline variant VCF, publicly available reference genome and transcriptome.

* Create a walk through explaining how to download all data needed, run regtools in various modes, parse results, visualize results (show screenshots of expected "answers"), etc.

**Comment by yanyangfeng** [comment]
Has this been done? If not, I could probably take this job to help acquaint myself with/think about the whole pipeline.

**Comment by malachig** [comment]
Nope, not done yet.  That would be great.  Maybe this could be done as a new section in the ReadTheDocs page.  A verbose explanation of each step and actual commands run on real data.  Then when we pull some interesting example out at the end we can discuss how to interpret them, produce a visualization, etc.

**Comment by malachig** [comment]
We do demonstrate basic use of regtools on RNAbio.org
